venues/MIDL.io/2019/Conference/webfield/areachairWebfieldAbstract.js

```diff
@@ -0,0 +1,699 @@
+
+// Assumes the following pattern for meta reviews and official reviews:
+// CONFERENCE + '/-/Paper' + number + '/Meta_Review'
```

Comment:
Is there a reason to overwrite the ac console here?

Reply:
The ACs are going to act as reviewers for the abstract submissions.

Reply:
so, you are setting the reviewer console in the ac console?